src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/Crucible.java

```diff
@@ -14,6 +14,7 @@
 import org.bukkit.World.Environment;
 import org.bukkit.block.Block;
 import org.bukkit.block.BlockFace;
+import org.bukkit.block.data.BlockData;
```

Comment:
Unused import

Reply:
Fixed, forgot to remove after experimentations